plugins/json_importer.cc

```diff
@@ -89,6 +90,13 @@ JSONImporterConfigDescription()
              "Row name expression for output dataset. Note that each row "
              "must have a unique name and that names cannot be objects.",
              SqlExpression::parse("lineNumber()"));
+    addAuto("arrays", &JSONImporterConfig::arrays,
+            "Describes how arrays are encoded in the JSON output.  For "
+            "''parse' (default), the arrays become structured values. "
```

Comment:
Not sure this is working.  Where is the conversion from string 'parse' to enum value 0 done?